lib/util/deprecation-warnings.ts

```diff
@@ -3,10 +3,10 @@ import * as _ from 'lodash';
 
 export = _.mapValues(
 	{
-		pubNubDeprecated: () =>
+		resinRenameDeprecation: () =>
```

Comment:
Ah, yes, that's because this PR is on top of #589, which does exactly that. This PR includes the changes from there, but only the 2nd commit matters (i.e. you can ignore this I think).